pkgs/development/python-modules/webgui-jupyter-widgets/default.nix

```
@@ -0,0 +1,85 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  jupyter-packaging,
+  jupyterlab,
+  notebook,
+  setuptools,
+  packaging,
+  yarnConfigHook,
+  fetchYarnDeps,
+  nodejs,
+  numpy,
+  jupyterlab-widgets,
+  ipywidgets,
+}:
+
+buildPythonPackage rec {
+  pname = "webgui-jupyter-widgets";
+  version = "0.2.31";
```

```suggestion
  version = "0.2.31";
```
Since this isn't using a tag this is an unstable version

---

by the way, the github source has been stopped for update for 11 months while the latest version on pypi is 0.2.37, seems like the author did not upgrade the github source, No where to find the lateset source code